examples/example_config_files/public_ivc_config.yaml

```diff
@@ -0,0 +1,169 @@
+name: Public Inverted Coax
+units:
+    length: mm
+    angle: deg
+    potential: V
+    temperature: K
+grid:
+  coordinates: cylindrical
+  axes:
+    r:
+      to: 40
+      boundaries: inf
+    phi:
+      from: 0
+      to: 0
+      boundaries: periodic
+    z:
+      from: -10
+      to: 90
+      boundaries:
+        left: inf
+        right: inf
+# grid:
+#     coordinates: cartesian
+#     axes:
+#         x:
+#             from: -40
+#             to: 40
+#             boundaries:
+#                 left: inf
+#                 right: inf
+#         y:
+#             from: -40
+#             to: 40
+#             boundaries:
+#                 left: inf
+#                 right: inf
+#         z:
+#             from: -10
+#             to: 90
+#             boundaries:
+#                 left: inf
+#                 right: inf
```

Comment:
Is it intended that this comment is still in here?

Reply:
Yes, so that I could tests both coordinate systems easily.
But we can change this again.

Reply:
No, it's fine.